src/environments/environment.ts

```diff
@@ -6,4 +6,5 @@
 export const environment = {
   production: false,
   distance: 32000,
+  googleMapsApiKey: 'AIzaSyCUvxTzvJ9kGEu2S7U641D7QGt5Xym0DjM',
```

Comment:
I got an email from [GitGuardian](https://gitguardian.com/) (which I had never heard of before) helpfully reminding me that this is a public repo and we probably shouldn't put our API keys here, especially the development key which has less restrictive referrer URLs. I'm going to deactivate this key and generate a new one, and come up with a new strategy for getting these keys into the projects (probably in `constants.ts`?)

Reply:
Resolved in https://github.com/azavea/climate-change-lab/pull/342/commits/2b384fb7e64e8cf2848ab0d0f3d3eee7f3146456